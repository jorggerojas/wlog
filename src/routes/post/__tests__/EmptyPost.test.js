// import React from "react";
// import { cleanup } from "@testing-library/react";
// import axiosMock from "axios";
// import EmptyPost from "../components/EmptyPost";
// import { renderWithRouter } from "../../../utils";

// afterEach(cleanup);
// jest.mock("axios");

test("a", () => {});
