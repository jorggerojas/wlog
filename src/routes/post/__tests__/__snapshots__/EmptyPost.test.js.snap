// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyPost/> component filling inputs right 1`] = `
<div
  class="sc-gsTCUz eBEudj uk-padding"
>
  <div>
    <h2
      class="sc-bZSQDF eMALkH uk-text-left"
    >
      Create here your post, fill all the fields and share!
    </h2>
    <form
      class="post uk-align-center"
    >
      <ul
        uk-accordion="multiple: true"
      >
        <li
          class="uk-open"
        >
          <a
            class="uk-accordion-title"
            href="/create"
          >
            <legend
              class="sc-idOhPF dXqleA"
            >
              Title
            </legend>
          </a>
          <div
            class="uk-accordion-content"
          >
            <div
              class="uk-margin uk-padding-small"
            >
              <input
                class="sc-cTkwdZ wKQwS uk-input"
                data-testid="title"
                name="title"
                placeholder="Something cool..."
                style="transform: translateX(0px) ;"
                type="text"
                value="My post title"
              />
            </div>
          </div>
        </li>
        <li>
          <a
            class="uk-accordion-title"
            href="/create"
          >
            <legend
              class="sc-idOhPF dXqleA"
            >
              Summary
            </legend>
          </a>
          <div
            class="uk-accordion-content"
          >
            <div
              class="uk-margin uk-padding-small"
            >
              <textarea
                class="sc-jNMdTA gHmUjv uk-textarea"
                data-testid="summary"
                name="summary"
                placeholder="Something short and cool"
                rows="3"
              >
                My summary
              </textarea>
            </div>
          </div>
        </li>
        <li>
          <a
            class="uk-accordion-title"
            href="/create"
          >
            <legend
              class="sc-idOhPF dXqleA"
            >
              Content
            </legend>
          </a>
          <div
            class="uk-accordion-content"
          >
            <div
              class="uk-margin uk-padding-small"
            >
              <textarea
                class="sc-jNMdTA gHmUjv uk-textarea"
                data-testid="content"
                name="content"
                placeholder="All you need is type"
                rows="10"
              >
                The content
              </textarea>
            </div>
          </div>
        </li>
        <li>
          <a
            class="uk-accordion-title"
            href="/create"
          >
            <legend
              class="sc-idOhPF dXqleA"
            >
              Keywords
            </legend>
          </a>
          <div
            class="uk-accordion-content"
          >
            <div
              class="uk-margin uk-padding-small"
            >
              <div>
                <p
                  class="uk-text-meta uk-text-small"
                >
                  Press the key
                   
                  <span
                    class="uk-text-italic"
                  >
                    "Space"
                  </span>
                   or
                  <span
                    class="uk-text-italic"
                  >
                    "," (comma)
                  </span>
                   or
                  <span
                    class="uk-text-italic"
                  >
                    "." (period)
                  </span>
                   
                  to add the word to the list.
                </p>
                <input
                  class="sc-cTkwdZ wKQwS uk-input"
                  data-testid="keywords"
                  placeholder="Few and cool words"
                  style="transform: translateX(0px) ;"
                  type="text"
                  value=""
                />
                <div
                  class="uk-placeholder uk-text-center"
                />
              </div>
            </div>
          </div>
        </li>
      </ul>
      <div
        class="uk-margin uk-padding-small uk-text-right"
      >
        <input
          class="sc-iUuytg eYORxA uk-button uk-button-primary"
          data-testid="save"
          type="submit"
          value="Save"
        />
      </div>
    </form>
  </div>
</div>
`;

exports[`<EmptyPost/> component filling inputs right 2`] = `
<div>
  <div
    class="sc-bdfBwQ jneblS uk-animation-fade"
  >
    <div
      uk-sticky="sel-target: .uk-navbar-container; cls-active: uk-navbar-sticky"
    >
      <nav
        class="sc-jSgupP coTTAM uk-navbar uk-navbar-container uk-margin"
      >
        <div
          class="uk-navbar-left"
        >
          <a
            class="uk-navbar-item uk-logo"
            href="/"
          >
            <span
              class="sc-dIUggk jYdzTb"
            >
              Wlog
            </span>
          </a>
        </div>
        <div
          class="uk-navbar-right"
        >
          <ul
            class="uk-navbar-nav"
          >
            <li
              class="uk-visible"
            >
              <div
                class="mid"
              >
                <label
                  class="rocker rocker-small"
                >
                  <input
                    type="checkbox"
                  />
                  <span
                    class="switch-left"
                  >
                    ðŸŒš
                  </span>
                  <span
                    class="switch-right"
                  >
                    ðŸŒž
                  </span>
                </label>
              </div>
            </li>
            <li
              class="uk-visible@s"
            >
              <a
                href="/user/RealUser1"
              >
                <span
                  class="sc-dIUggk jYdzTb"
                >
                  REALUSER1
                </span>
              </a>
            </li>
            <li
              class="uk-hidden@s"
            >
              <a
                href="/user/RealUser1"
                uk-icon="icon:user; ratio:1.3"
              />
            </li>
          </ul>
        </div>
        <div
          class="uk-navbar-center"
        >
          <a
            class="uk-navbar-toggle"
            href="/create"
          >
            <span
              class="white"
              uk-icon="icon:plus-circle"
            />
          </a>
        </div>
      </nav>
    </div>
    <div
      class="sc-gsTCUz eBEudj uk-padding"
    >
      <div>
        <h2
          class="sc-bZSQDF eMALkH uk-text-left"
        >
          Create here your post, fill all the fields and share!
        </h2>
        <form
          class="post uk-align-center"
        >
          <ul
            uk-accordion="multiple: true"
          >
            <li
              class="uk-open"
            >
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Title
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <input
                    class="sc-cTkwdZ wKQwS uk-input"
                    data-testid="title"
                    name="title"
                    placeholder="Something cool..."
                    style="transform: translateX(0px) ;"
                    type="text"
                    value="My post title"
                  />
                </div>
              </div>
            </li>
            <li>
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Summary
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <textarea
                    class="sc-jNMdTA gHmUjv uk-textarea"
                    data-testid="summary"
                    name="summary"
                    placeholder="Something short and cool"
                    rows="3"
                  >
                    My summary
                  </textarea>
                </div>
              </div>
            </li>
            <li>
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Content
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <textarea
                    class="sc-jNMdTA gHmUjv uk-textarea"
                    data-testid="content"
                    name="content"
                    placeholder="All you need is type"
                    rows="10"
                  >
                    The content
                  </textarea>
                  <div
                    class="uk-text-danger uk-text-bold"
                  >
                    Content is required
                  </div>
                </div>
              </div>
            </li>
            <li>
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Keywords
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <div>
                    <p
                      class="uk-text-meta uk-text-small"
                    >
                      Press the key
                       
                      <span
                        class="uk-text-italic"
                      >
                        "Space"
                      </span>
                       or
                      <span
                        class="uk-text-italic"
                      >
                        "," (comma)
                      </span>
                       or
                      <span
                        class="uk-text-italic"
                      >
                        "." (period)
                      </span>
                       
                      to add the word to the list.
                    </p>
                    <input
                      class="sc-cTkwdZ wKQwS uk-input"
                      data-testid="keywords"
                      placeholder="Few and cool words"
                      style="transform: translateX(0px) ;"
                      type="text"
                      value=""
                    />
                    <div
                      class="uk-placeholder uk-text-center"
                    />
                  </div>
                </div>
              </div>
            </li>
          </ul>
          <div
            class="uk-margin uk-padding-small uk-text-right"
          >
            <input
              class="sc-iUuytg eYORxA uk-button uk-button-primary"
              data-testid="save"
              type="submit"
              value="Save"
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;

exports[`<EmptyPost/> component render component with session 1`] = `
<div>
  <div
    class="sc-bdfBwQ jneblS uk-animation-fade"
  >
    <div
      uk-sticky="sel-target: .uk-navbar-container; cls-active: uk-navbar-sticky"
    >
      <nav
        class="sc-jSgupP coTTAM uk-navbar uk-navbar-container uk-margin"
      >
        <div
          class="uk-navbar-left"
        >
          <a
            class="uk-navbar-item uk-logo"
            href="/"
          >
            <span
              class="sc-dIUggk jYdzTb"
            >
              Wlog
            </span>
          </a>
        </div>
        <div
          class="uk-navbar-right"
        >
          <ul
            class="uk-navbar-nav"
          >
            <li
              class="uk-visible"
            >
              <div
                class="mid"
              >
                <label
                  class="rocker rocker-small"
                >
                  <input
                    type="checkbox"
                  />
                  <span
                    class="switch-left"
                  >
                    ðŸŒš
                  </span>
                  <span
                    class="switch-right"
                  >
                    ðŸŒž
                  </span>
                </label>
              </div>
            </li>
            <li
              class="uk-visible@s"
            >
              <a
                href="/user/RealUser1"
              >
                <span
                  class="sc-dIUggk jYdzTb"
                >
                  REALUSER1
                </span>
              </a>
            </li>
            <li
              class="uk-hidden@s"
            >
              <a
                href="/user/RealUser1"
                uk-icon="icon:user; ratio:1.3"
              />
            </li>
          </ul>
        </div>
        <div
          class="uk-navbar-center"
        >
          <a
            class="uk-navbar-toggle"
            href="/create"
          >
            <span
              class="white"
              uk-icon="icon:plus-circle"
            />
          </a>
        </div>
      </nav>
    </div>
    <div
      class="sc-gsTCUz eBEudj uk-padding"
    >
      <div>
        <h2
          class="sc-bZSQDF eMALkH uk-text-left"
        >
          Create here your post, fill all the fields and share!
        </h2>
        <form
          class="post uk-align-center"
        >
          <ul
            uk-accordion="multiple: true"
          >
            <li
              class="uk-open"
            >
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Title
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <input
                    class="sc-cTkwdZ wKQwS uk-input"
                    data-testid="title"
                    name="title"
                    placeholder="Something cool..."
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </li>
            <li>
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Summary
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <textarea
                    class="sc-jNMdTA gHmUjv uk-textarea"
                    data-testid="summary"
                    name="summary"
                    placeholder="Something short and cool"
                    rows="3"
                  />
                </div>
              </div>
            </li>
            <li>
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Content
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <textarea
                    class="sc-jNMdTA gHmUjv uk-textarea"
                    data-testid="content"
                    name="content"
                    placeholder="All you need is type"
                    rows="10"
                  />
                </div>
              </div>
            </li>
            <li>
              <a
                class="uk-accordion-title"
                href="/create"
              >
                <legend
                  class="sc-idOhPF dXqleA"
                >
                  Keywords
                </legend>
              </a>
              <div
                class="uk-accordion-content"
              >
                <div
                  class="uk-margin uk-padding-small"
                >
                  <div>
                    <p
                      class="uk-text-meta uk-text-small"
                    >
                      Press the key
                       
                      <span
                        class="uk-text-italic"
                      >
                        "Space"
                      </span>
                       or
                      <span
                        class="uk-text-italic"
                      >
                        "," (comma)
                      </span>
                       or
                      <span
                        class="uk-text-italic"
                      >
                        "." (period)
                      </span>
                       
                      to add the word to the list.
                    </p>
                    <input
                      class="sc-cTkwdZ wKQwS uk-input"
                      data-testid="keywords"
                      placeholder="Few and cool words"
                      type="text"
                      value=""
                    />
                    <div
                      class="uk-placeholder uk-text-center"
                    />
                  </div>
                </div>
              </div>
            </li>
          </ul>
          <div
            class="uk-margin uk-padding-small uk-text-right"
          >
            <input
              class="sc-iUuytg eYORxA uk-button uk-button-primary"
              data-testid="save"
              type="submit"
              value="Save"
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;

exports[`<EmptyPost/> component render component without session 1`] = `
<div>
  <div
    class="sc-eCssSg fEobmb uk-animation-fade uk-padding-large"
  >
    <div>
      <img
        alt="not found"
        class="sc-hKgILt cNbHLb"
        src="search.svg"
      />
      <h3
        class="sc-dmlrTW jDCTgQ uk-margin-small"
      >
        We're looking for 
        <code
          class="sc-kfzAmx AFNMD"
        >
          /create
        </code>
         but we don't have any results
      </h3>
    </div>
    <div
      class="uk-margin"
    >
      <a
        href="/"
      >
        <code
          class="sc-kfzAmx AFNMD"
        >
          Return to the main page
        </code>
      </a>
    </div>
  </div>
</div>
`;
